packages/cart/integration-tests/__tests__/services/cart-module/index.spec.ts

```diff
@@ -2332,4 +2333,184 @@ describe("Cart Module Service", () => {
       expect(taxLines?.length).toBe(0)
     })
   })
+
+  describe.only("retrieve with totals decoration", () => {
```

Comment:
todo: rm only

Reply:
Will be removed once approach is approved.